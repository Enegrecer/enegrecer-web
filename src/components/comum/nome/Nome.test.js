import React from 'react';
import toJson from 'enzyme-to-json';
import { mount } from 'enzyme';
import Nome from './index';

describe('Nome', () => {
  it('deve seguir a estrutura definida ', () => {
    const nome = mount(<Nome id={'3'} />);
    const nomeJson = toJson(nome);
    expect(nomeJson).toMatchSnapshot();
  });
});

