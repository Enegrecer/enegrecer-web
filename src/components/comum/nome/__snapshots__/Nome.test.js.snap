// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nome deve seguir a estrutura definida  1`] = `
<nome
  divClasse="col s12 m6 l6"
  id="3"
  onChange={[Function]}
  state=""
  validacoes={Array []}
>
  <campoTexto
    divClasse="input-field col s12 m6 l6"
    id="3"
    label="Nome (m치ximo de 40 caracteres)"
    maxLen={40}
    onChange={[Function]}
    placeholder="Nome (m치ximo de 40 caracteres)"
    state=""
    type="text"
    validacoes={
      Array [
        [Function],
        [Function],
        [Function],
      ]
    }
  >
    <div
      className="input-field col s12 m6 l6"
    >
      <input
        id="3"
        maxLength={40}
        placeholder="Nome (m치ximo de 40 caracteres)"
        type="text"
      />
      <label
        className="active"
        data-success="right"
        htmlFor="3"
      >
        Nome (m치ximo de 40 caracteres)
      </label>
    </div>
  </campoTexto>
</nome>
`;
